mysql/check.py

```diff
@@ -1008,24 +1014,53 @@ def _get_stats_from_innodb_status(self, db):
                 results['Innodb_os_file_writes'] = long(row[4])
                 results['Innodb_os_file_fsyncs'] = long(row[8])
             elif line.find('Pending normal aio reads:') == 0:
-                # Pending normal aio reads: 0, aio writes: 0,
-                # or Pending normal aio reads: 0 [0, 0] , aio writes: 0 [0, 0] ,
-                # or Pending normal aio reads: [0, 0, 0, 0] , aio writes: [0, 0, 0, 0] ,
-                # or Pending normal aio reads: 0 [0, 0, 0, 0] , aio writes: 0 [0, 0, 0, 0] ,
-                if len(row) == 14:
-                    results['Innodb_pending_normal_aio_reads'] = long(row[4])
-                    results['Innodb_pending_normal_aio_writes'] = long(row[10])
+                if len(row) == 8:
+                    # (len(row) == 8)  Pending normal aio reads: 0, aio writes: 0,
+                    if row[4].isdigit() and row[7].isdigit():
```

Comment:
The only issue of doing the entire surrounding try/except is that we are coming to the point where multiple len==16 return different output. So we would have to check if some values were a digit anyway. (Also this code came from @derekwbrown after our discussion, can't take the credit :) )